packages/account-fs/people/search.js

```diff
@@ -112,6 +112,12 @@ export const SearchCapability = {
         filteredContacts.push(person)
         continue
       }
+
+      if (person.EMAIL && person.EMAIL.some(email => email.toLowerCase().includes(queryString))) {
```

Comment:
`person.Email.some` is breaking for accounts with existing data. For new accounts it works fine
<img width="588" alt="Screenshot 2024-04-15 at 12 24 13 PM" src="https://github.com/shovelers/network-monorepo/assets/8470415/a7480cf3-d7ae-4a0c-9024-0798b10a1686">

Reply:
have resolved this. do have a look at it